db/app.rb

```diff
@@ -0,0 +1,26 @@
+require 'sinatra'
+require 'sinatra/reloader'
+require "bundler/setup"
+require "pg"
+require "active_record"
+require "pry"
+
+require_relative "db/connection"
+require_relative "models/apartment"
+require_relative "models/tenant"
+
+get '/' do
+  erb:":apartments"
+end
+
+get '/apartments' do
+  erb:"apartments/1"
+end
```

Comment:
really the erb should be called `apartments/show` the route itself should be `get '/apartments/:is`